### DISCUSSION TITLE
how big can the CopyQ .dat file be

### DISCUSSION THREAD

**Discussion by marceepoonu** [hluk/CopyQ#2159] [Q&A]
I just started using CopyQ.  I used to use Clipmate.
Based on the manual I read, I get the impression that the items are stored in a .dat file.  How big can the CopyQ .dat file be?
Is there a way to archive the data, before a .dat file gets so large it becomes slow or prone to corruption, but still be able to search through it for old clip items?

**Comment by laur89** [comment]
Think you're approaching this from the wrong end. It'd be difficult to control the size and then consolidate the dat file. Instead you should limit the number of elements stored:  `Preferences -> History -> Maximum number of items in history`
